x/interchainstaking/keeper/receipt.go

```diff
@@ -248,6 +248,9 @@ func (k *Keeper) SubmitTx(ctx sdk.Context, msgs []sdk.Msg, account *types.ICAAcc
 		return sdkioerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
 
+	// TO DO: discuss on adding this because we are receiving err capability not found from scoppedICAController
```

Comment:
is this TODO: addressed?

Reply:
not yet, we are discussing it in slack channel